deployment/GI05MOAS-PG515_Deploy.csv

```diff
@@ -1,8 +1,8 @@
 CUID_Deploy,deployedBy,CUID_Recover,recoveredBy,Reference Designator,deploymentNumber,versionNumber,startDateTime,stopDateTime,mooring.uid,node.uid,sensor.uid,lat,lon,orbit,deployment_depth,water_depth,notes
-AR35-05,,,,GI05MOAS-PG515-00-ENG000000,3,1,,,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00515,CGCON-GLDRGP-00515,,,,200,200,
-AR35-05,,,,GI05MOAS-PG515-01-CTDGVM000,3,1,,,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00515,CGINS-CTDGVM-09231,,,,200,200,
-AR35-05,,,,GI05MOAS-PG515-02-DOSTAM000,3,1,,,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00515,CGINS-DOSTAM-00414,,,,200,200,
-AR35-05,,,,GI05MOAS-PG515-03-FLORTM000,3,1,,,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00515,CGINS-FLORTM-03501,,,,200,200,
-AR35-05,,,,GI05MOAS-PG515-04-FLORTO000,3,1,,,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00515,CGINS-FLORTO-01239,,,,200,200,
-#AR35-05,,,,GI05MOAS-PG515-05-NUTNRM000,3,1,,,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00515,CGINS-NUTNRM-00760,,,,200,200,need vendor cal doc to create csv
-AR35-05,,,,GI05MOAS-PG515-06-PARADM000,3,1,,,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00515,CGINS-PARADM-50185,,,,200,200,
+AR35-05,,AR35-05,,GI05MOAS-PG515-00-ENG000000,3,1,2019-08-06T17:20:00,2019-08-06T21:28:00,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00515,CGCON-GLDRGP-00515,59.9162,-39.26579,,200,200,glider leaked
+AR35-05,,AR35-05,,GI05MOAS-PG515-01-CTDGVM000,3,1,2019-08-06T17:20:00,2019-08-06T21:28:00,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00515,CGINS-CTDGVM-09231,59.9162,-39.26579,,200,200,glider leaked
+AR35-05,,AR35-05,,GI05MOAS-PG515-02-DOSTAM000,3,1,2019-08-06T17:20:00,2019-08-06T21:28:00,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00515,CGINS-DOSTAM-00414,59.9162,-39.26579,,200,200,glider leaked
+AR35-05,,AR35-05,,GI05MOAS-PG515-03-FLORTM000,3,1,2019-08-06T17:20:00,2019-08-06T21:28:00,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00515,CGINS-FLORTM-03501,59.9162,-39.26579,,200,200,glider leaked
+AR35-05,,AR35-05,,GI05MOAS-PG515-04-FLORTO000,3,1,2019-08-06T17:20:00,2019-08-06T21:28:00,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00515,CGINS-FLORTO-01239,59.9162,-39.26579,,200,200,glider leaked
+AR35-05,,AR35-05,,GI05MOAS-PG515-05-NUTNRM000,3,1,2019-08-06T17:20:00,2019-08-06T21:28:00,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00515,CGINS-NUTNRM-00760,59.9162,-39.26579,,200,200,glider leaked
+AR35-05,,AR35-05,,GI05MOAS-PG515-06-PARADM000,3,1,2019-08-06T17:20:00,2019-08-06T21:28:00,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00515,CGINS-PARADM-50185,59.9162,-39.26579,,200,200,glider leaked
```

Comment:
I'd like to get confirmation from @cdobs on what the note should say.  I believe it is an "aft glider leak"

Reply:
Roger that - table just stated 'leak detect'.  Will wait for his input.

Reply:
Collin says "glider leaked" is fine.

Reply:
Looks good.